doc/backend_vars.asciidoc

```diff
@@ -171,11 +171,13 @@ QEMU_SMBIOS;see qemu -smbios ?;undef;pass this value to qemu -smbios
 QEMU_SOUNDHW;see qemu -soundhw ?;had;pass this value to qemu -soundhw (for qemu < 4.2)
 QEMU_AUDIODEV;see qemu -device ?;intel-hda;Audio device to use with audiodev to qemu -device (for qemu >= 4.2)
 QEMU_AUDIOBACKEND;see qemu -audio-help;none;Audio backend to use with audiodev (for qemu >= 4.2)
-QEMU_COMPRESS_LEVEL;integer;6;Sets the compression level used for memory dumps and snapshots. Zero turns compression off and 9 is the maximum level. Generally there is little improvement in compression ratio by increasing the level, but the CPU time can be high on some platforms.
-QEMU_COMPRESS_THREADS;integer;QEMUCPUS;Number of threads used for compressing memory dumps and snapshots.
+QEMU_COMPRESS_LEVEL;integer;6;Sets the compression level used for memory dumps and snapshots. Zero turns compression off and 9 is the maximum level. Generally there is little improvement in compression ratio by increasing the level, but the CPU time can be high on some platforms. (For qemu < 9.1.0)
+QEMU_COMPRESS_THREADS;integer;QEMUCPUS;Number of threads used for compressing memory dumps and snapshots. (For qemu < 9.1.0)
 QEMU_NON_FATAL_DBUS_CALL;boolean;0;Ignore failed dbus calls and ignore instead of fatal exits
 QEMU_MAX_BANDWIDTH;integer;INT_MAX;Limits the transfer rate during a snapshot.
 QEMU_DUMP_COMPRESS_METHOD;string;xz;The compression to use during a memory dump. Can be set to xz, bzip2 or internal (QEMU's internal compression, not compatible with crash or gdb). If xz is set, but not available, it will fallback to bzip2. Also see QEMU_COMPRESSION_LEVEL.
+QEMU_MULTIFD_CHANNELS;integer;2;Number of channels used to migrate data in parallel. This is the same number that the number of sockets used for migration. Can be set from 1 to 255.
+QEMU_USE_MULTIFD_FOR_MIGRATION;boolean;0;Force use of multifd for migration. Usefull only for qemu < 9.1.0, since it is the mandatory for qemu >= 9.1.0.
```

Comment:
Do you think we can abstract this so that we reuse the old variables? Something like "compress level" and "compress threads" I would understand as os-autoinst user. The new variables are very qemu specific knowledge and also breaking compatibility.

Reply:
As it is a very different meaning, I would not vote for keeping the same vars.

Reply:
I know that they have different meaning. That is why I was wondering how we could abstract this to not break any test definitions that rely on this. Or are those merely internal configuration settings that don't have a significant impact on tests? If that's the case then I suggest to at least rephrase "since it is the mandatory for qemu >= 9.1.0". Does it mean "implicit for qemu >= 9.1"?

Reply:
Right, there are defaults value and those settings can be omitted. I will replace `mandatory` with `implicit`.

Reply:
Removed `QEMU_USE_MULTIFD_FOR_MIGRATION`